### DISCUSSION TITLE
is there a way to restrict reordering grid list items to just one row at a time?

### DISCUSSION THREAD

**Discussion by binaryartifex** [adobe/react-spectrum#6576] [Q&A]
with my use case, there is no relationship between selected items and the drag functionality. selected items means which table columns are visible, order of items reflects order of table columns, however when more than one are selected im forced to drag all of them in aggregate. is there a way to avoid this?

![image](https://github.com/adobe/react-spectrum/assets/68464707/66dd9999-8794-4f32-bf32-65c15cfe4de1)

**Comment by snowystinger** [comment]
Could you use GridList instead? Then I think you could create different checkboxes which aren't selection in the usual sense. You could also give that column of checkboxes a different name, "visible columns" or something like that.

**Reply by rothsandro** [reply]
I have the exact same use case. A regular checkbox without enabling selection on the GridList works but lacks certain selection features like Space to select and range selection with Shift + Click.

There's also this feature request: #7449
